### PR TITLE
Add regen warning to getting started

### PR DESCRIPTION
**Pull request by iq-fred**

### CODE REVIEW

File: docs/tutorials/testing_with_control_center.rst [hunk 1]
```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators.
```
Comment (vertiq-jordan) [inline docs/tutorials/testing_with_control_center.rst:36]: I think you should specify what *aggressive commands* are. We know that the biggest problem comes from people stopping hard, but people reading this won't. Even if it is wordier, I think it is best to be as clear as possible about what can cause the issues.

File: docs/tutorials/testing_with_control_center.rst [hunk 2]
```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators. 
+  Power supplies generally cannot absorb the energy generated by the module, while batteries can, which can lead to dangerous voltage spikes when connected to a power supply.
```
Comment (vertiq-jordan) [inline docs/tutorials/testing_with_control_center.rst:37]: In general, power supplies, unlike batteries, cannot absorb the energy generated by the module. As such, aggressive commands can lead ...

File: docs/tutorials/testing_with_control_center.rst [hunk 3]
```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators. 
+  Power supplies generally cannot absorb the energy generated by the module, while batteries can, which can lead to dangerous voltage spikes when connected to a power supply. 
+  To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the regeneration voltage protection feature.
```
Comment (vertiq-jordan) [inline docs/tutorials/testing_with_control_center.rst:38]: link to the regen voltage protection feature? Not sure if we have one

Comment (iq-fred) [reply docs/tutorials/testing_with_control_center.rst:38]: Sadly, we do not have any documentation to link to on it. Hopefully we will in the future

**Review by vertiq-jordan — APPROVED** [review]

**Pull request merged**